spec/components/previews/login_button_component_preview.rb

```diff
@@ -18,4 +21,28 @@ def workbench(
       ),
     )
   end
+
+  private
+
+  def css_file_path
+    Rails.root.join(
+      'app',
+      'assets',
+      'builds',
+      'login_button_component.css',
+    )
+  end
+
+  def css
+    File.read(css_file_path)
+  end
+
+  def inject_style(html)
+    <<~HTML
+      <style>
+        #{css}
+      </style>
+      #{html}
+    HTML
+  end
```

Comment:
since `Rails.root.join` returns a `Pathname` object (which has `#read`) we could probably remove one of these methods
```suggestion
  def inject_style(html)
    <<~HTML
      <style>
        #{css_file_path.read}
      </style>
      #{html}
    HTML
  end
```